distro/wildfly/pom.xml

```diff
@@ -14,6 +14,21 @@
   <description>${project.name}</description>
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>jakarta.platform</groupId>
+        <artifactId>jakarta.jakartaee-bom</artifactId>
+        <version>${version.jakarta-ee}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.glassfish.jersey</groupId>
+        <artifactId>jersey-bom</artifactId>
+        <version>3.1.11</version>
```

Comment:
jersey-bom is defined in internal-dependencies with version 4.0.0 by property version.jersey3